doc/src/lrtable.md

```diff
@@ -1,8 +1,10 @@
 # `lrtable`
 
-`lrtable` takes in grammars from [`cfgrammar`](cfgrammar.html) and creates LR
-state tables from them. Few users will be interested in its functionality
-directly, except those doing advanced forms of grammar analysis.
+`lrtable` ([crate](https://crates.io/crates/lrpar);
```

Comment:
The first link is wrong. It should link to lrtable not lrpar.

Reply:
Oops! Fixed in https://github.com/softdevteam/grmtools/pull/170/commits/191b82f352758375653874cb32d4ae1ad29929cd.